h_periodic/beat.py

```diff
@@ -8,26 +8,32 @@
 celery.conf.update(
     beat_schedule={
         "purge-deleted-annotations": {
+            "options": {"expires": 5},
             "task": "h.tasks.cleanup.purge_deleted_annotations",
             "schedule": timedelta(hours=1),
```

Comment:
I wonder if we want to tune the expiry to the schedule time. For example, if we have 1 hour until the next time, we could probably afford to wait 30 minutes here. This would allow us to ride over a short outage for celery without dropping these long running tasks quite as quickly.

Reply:
I've changed these to expire after half the time until the next scheduled task. So after 30 mins for hourly tasks etc